docs/source/native-delegates-executorch-arm-delegate.md

```diff
@@ -0,0 +1,3 @@
+# ARM Backend
```

Comment:
ARM -> Arm EthosU Backend

ARM is too broad... Is it Cortex-A, Cortex-M? 

In this case, we're only talking about running on Cortex Ethos-U backend for acceleration.